### PR TITLE
Dashboard layout improvements

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: src/controllers/dashboardpage.js [hunk 1]
```diff
@@ -12,27 +12,23 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
...
-            } else if (isTranscode) {
+            } else if (displayPlayMethod === "Transcode") {
                 title = globalize.translate("Transcoding");
                 text.push(globalize.translate("MediaIsBeingConverted"));
-
                 if (session.TranscodingInfo && session.TranscodingInfo.TranscodeReasons && session.TranscodingInfo.TranscodeReasons.length) {
                     text.push("<br/>");
                     text.push(globalize.translate("LabelReasonForTranscoding"));
                     session.TranscodingInfo.TranscodeReasons.forEach(function (transcodeReason) {
-                        text.push(globalize.translate("" + transcodeReason));
```
Comment (JustAMan) [inline src/controllers/dashboardpage.js]: why change? this looks like a stringification safety net to me

File: src/controllers/dashboardpage.js [hunk 2]
```diff
@@ -398,13 +395,12 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
...
             if (showTranscodingInfo) {
                 var line = [];
-
                 if (session.TranscodingInfo) {
                     if (session.TranscodingInfo.Bitrate) {
                         if (session.TranscodingInfo.Bitrate > 1e6) {
                             line.push((session.TranscodingInfo.Bitrate / 1e6).toFixed(1) + " Mbps");
                         } else {
-                            line.push(Math.floor(session.TranscodingInfo.Bitrate / 1e3) + " kbps");
+                            line.push(Math.floor(session.TranscodingInfo.Bitrate / 1e3) + " Kbps");
```
Comment (JustAMan) [inline src/controllers/dashboardpage.js]: I understand that per ISO 1 Kb is 1000 bytes, but maybe move to 1 KiB = 1024 bytes here somehow?..

Comment (dkanada) [reply src/controllers/dashboardpage.js]: Generally Kbps is used for any kind of streaming measurement which is why I kept the label the same as before.

File: src/controllers/dashboardpage.js (hunk 1 above)
Comment (dkanada) [reply src/controllers/dashboardpage.js]: Javascript is usually very good about casting types and I didn't notice any issues when I was testing this pull request.

**Comment by dkanada** [comment]
@grafixeyehero that's definitely an issue but I didn't add that functionality. It already existed, someone just added a toggle in the user settings.

**Comment by grafixeyehero** [comment]
OK Merge it

**Comment by grafixeyehero** [comment]
Fix toggle https://github.com/jellyfin/jellyfin-web/pull/466

**Pull request merged**